src/main/java/com/gruzini/tennistico/controllers/DashboardController.java

```diff
@@ -15,15 +18,18 @@
 public class DashboardController {
 
     private final NotificationService notificationService;
+    private final UserService userService;
 
-    public DashboardController(NotificationService notificationService) {
+    public DashboardController(final NotificationService notificationService, final UserService userService) {
         this.notificationService = notificationService;
+        this.userService = userService;
     }
 
     @GetMapping
-    public String showDashboard(final Model model, final Principal principal){
+    public String showDashboard(final Model model, final Principal principal, final HttpSession session){
         final List<NotificationDto> notifications = notificationService.getByUser(principal.getName());
         model.addAttribute("notifications", notifications);
+        session.setAttribute("loggedUser", userService.getByEmail(principal.getName()));
```

Comment:
dodajesz całego Usera? trochę słabo, nie uważasz? Tam jest jego mail, hasło itp. Może spróbujemy dodać jakieś małe Dto z fullName i points? albo nawet dwa atrybuty osobno?

edit: zobacz co niżej

Reply:
To akurat bardzo słuszna uwaga.

Reply:
good